mappings/net/minecraft/entity/passive/FrogEntity.mapping

```diff
@@ -0,0 +1,26 @@
+CLASS net/minecraft/unmapped/C_gcfircge net/minecraft/entity/passive/FrogEntity
+	FIELD f_aydfmrgq floatAnimationState Lnet/minecraft/unmapped/C_kxntavoz;
+	FIELD f_fhflvgeh MODULES Lcom/google/common/collect/ImmutableList;
+	FIELD f_foqpymcy TYPE Lnet/minecraft/unmapped/C_rinmcaxy;
+	FIELD f_hdhfqunh tongueAnimationState Lnet/minecraft/unmapped/C_kxntavoz;
+	FIELD f_ituaqaqf SENSORS Lcom/google/common/collect/ImmutableList;
+	FIELD f_kbrwwixv longJumpAnimationState Lnet/minecraft/unmapped/C_kxntavoz;
+	FIELD f_lpztiyuj SLIME_INGREDIENT Lnet/minecraft/unmapped/C_tcpsydrv;
+	FIELD f_mupmmsbd EATING_ENTITY_ID Lnet/minecraft/unmapped/C_rinmcaxy;
```

Comment:
this is a new one, weird. Why is there a need for an entity ID regarding eating?

Reply:
Ill have to look at this after my AP Studying (4 day study and work period I have been talking about a lot) I am quite literally reviewing this on my 5 minute break.

Reply:
This refers to the id for the "tongue target" entry in the entity's data (mojang calls it DATA_TONGUE_TARGET_ID).

Reply:
got it